app/helpers/routing_helper.rb

```diff
@@ -0,0 +1,54 @@
+require 'httparty'
+require 'json'
+
+module RoutingHelper
+  def self.seconds_to_minsec(sec)
+    format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
+  end
+
+  def self.valid_coordinates(coordinates)
+    return false if coordinates.blank?
+
+    regex = /^[0-9]{1,2}(\.[0-9]{1,6})?,[0-9]{1,2}(\.[0-9]{1,6})?$/
+    coordinates.match(regex)
+  end
+
+  def self.calculate_route(start, destination)
+    return nil unless valid_coordinates(start) && valid_coordinates(destination) # No route requested
+
+    start_location = start.split(",")
+    dest_location = destination.split(",")
+    # The API requires long, lat instead of lat, long so we have to swap those
```

Comment:
We could unpack the tuple right here at the assignment to get rid of the comment (the ordering of lat and long would be right in the url)